lib/airbrake/rack/notice_builder.rb

```diff
@@ -62,7 +62,7 @@ def add_context(notice)
           context[:action] = @controller.action_name
         end
 
-        notice[:context].merge!(@user.to_hash) if @user
+        notice[:context].merge!(@user.data) if @user
```

Comment:
I like your thinking, Kyrylo, but I think the `to_hash` convention would be quicker for the reader to understand, in a Ruby context.